### ISSUE TITLE
trying to push an image to GCR

### ISSUE THREAD

**Issue by prajathr** [Shippable/support#3761]
trying to build a docker image and push it to gcr. Have all necessary integrations in place. Image push is not happening. The yml file is available at https://github.com/prajathr/os-wp-demo/blob/master/shippable.yml. can you please guide in the right direction.

**Comment by manishas** [comment]
@prajathr can you try indenting your yml so that `post_ci` has the same level of indentation as the `ci` section?

I think your yml isn't being parsed as expected due to this issue. The parsed version of the yml is available [here](https://app.shippable.com/github/prajathr/os-wp-demo/runs/7/1/yml) and the build section doesn't even show the `post_ci` part due to the parsing error.

**Comment by prajathr** [comment]
@manishas your suggestion worked. Now i get an error at docker build. "unable to prepare context: unable to evaluate symlinks in Dockerfile path: lstat /root/src/github.com/prajathr/os-wp-demo/Dockerfile: no such file or directory". I understand this is because the file is missing. But does this file need to be a part of the git repo.

**Comment by rageshkrishna** [comment]
@prajathr Yes. If you are building a docker image, your Dockerfile needs to be part of the repository.
